lib/FrontendTool/FrontendTool.cpp

```diff
@@ -819,25 +824,26 @@ static bool performCompile(CompilerInstance &Instance,
         // once for each such input.
         for (auto *PrimaryFile : Instance.getPrimarySourceFiles()) {
           auto SM = performSILGeneration(*PrimaryFile, SILOpts, None);
-          SMs.push_back(std::make_pair(std::move(SM), !fileIsSIB(PrimaryFile)));
+          PSGIs.push_back(PostSILGenInputs{
+              std::move(SM), !fileIsSIB(PrimaryFile), PrimaryFile});
```

Comment:
The `fileIsSIB` check is definitely going to be `false` because SIB files are a kind of SerializedASTFile, not a SourceFile.

Reply:
Good catch. Replace !fileIsSIB(...) with "true".